defaults/main.yml

```diff
@@ -4,30 +4,34 @@
 minio_server_bin: /usr/local/bin/minio
 minio_client_bin: /usr/local/bin/mc
 
-# Runtime user and group for the minio server service
+# Runtime user and group for the Minio server service
 minio_user: minio
 minio_group: minio
 
-# Path to the file containing the ENV variables for the minio server
+# Path to the file containing the ENV variables for the Minio server
 minio_server_envfile: /etc/default/minio
 
 # Minio server listen address
 minio_server_addr: ":9091"
 
 # Minio server data directories
-minio_server_datadirs: [ ]
+minio_server_datadirs:
+  - /usr/share/minio
```

Comment:
According to [hier(7)](http://man7.org/linux/man-pages/man7/hier.7.html) /usr/share directory is used to:
> This directory contains subdirectories with specific application data, that can be shared among different architectures of the same OS. Often one finds stuff here that used to live in /usr/doc or /usr/lib or /usr/man.

Maybe it would be better to use sth like `/var/lib/minio` ?

Reply:
Makes perfectly sense.
I am going to push it with the fixes for the CI :)